recipe/meta.yaml

```diff
@@ -165,3 +160,5 @@ extra:
     - ericpre
     - douglowe
     - tschoonj
+    - apryor6
```

Comment:
@apryor6, can you confirm that you would be willing to be a maintainer? In staged-recipes, you did mention that you will be happy to follow this! :)